wsbroadcastserver/dictionary.go

```diff
@@ -0,0 +1,18 @@
+package wsbroadcastserver
+
+import "compress/flate"
+
+const DeflateCompressionLevel = flate.BestCompression
+
+func GetStaticCompressorDictionary() []byte {
```

Comment:
How will rollout of a dictionary upgrade work?

Reply:
The simplest way would be to change the `FeedServerVersion` with the upgrade making it a breaking change for old client version. Backward compatible solution could be achieved with slight adjustment of how server version is checked by the client.

Reply:
Just repeating some of the points from standup: We shouldn't need to upgrade the dictionary all the time (or at all) so it seems fine to bundle it together with a consensus upgrade. Otherwise we could update the version handling logic to allow older client versions and change behavior based on that, but given that we have a path forward it's not worth the complexity.